Dockerfile

```diff
@@ -1,15 +1,25 @@
-FROM golang:1.19.8-alpine as builder
-RUN apk add gcc musl-dev linux-headers git bash curl
-RUN git clone https://github.com/AthanorLabs/atomic-swap
+FROM golang:latest as builder
```

Comment:
should we pin the version to 1.20?

Reply:
It is up to you. There is a small, but non-zero, risk that our code stops building or working when 1.21 is released. In which case, we'd want to fix it as quickly as possible.  Locking it would prevent that risk, but also mean that we need to update the Dockerfile on every release. It would be nice if we could get CI to automatically move to the lastest stable/released version of go.  I didn't look into it when creating the 1.20 go upgrade PR.

Reply:
I pinned it to 1.20, since I was making one more edit to the file anyways.